debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorTask.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle;
+
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.stream.Collectors;
+
+import org.apache.kafka.connect.source.SourceRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import io.debezium.connector.base.ChangeEventQueue;
```

Comment:
Does not `ChangeEventQueue`  belong to `pipeline` package?

Reply:
Yes, the package structuring needs another pass overall. Didn't spend too much time on it yet. Probably makes sense to move this now.